src/.vuepress/styles/index.styl

```diff
@@ -20,5 +25,184 @@ html {
+}
+
+header.navbar {
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+    padding-left: 5rem;
+    height: auto;
+    line-height: 6.5rem;
+}
+
+header.navbar .links {
+    padding-right: 3rem;
+    top: 0.6rem;
+}
+
+.nav-item a {
+    // font-weight: 1000;
+    font-family: 'Inter-Bold', -apple-system, sans-serif;
+    font-size: 18px;
+}
+
+@media screen and (min-width: 1179px) { 
+        .nav-links {
+        padding: 0 3rem !important;
+        transition: all 0.2s ease;
+    }
+}
+
+.nav-links {
+    padding: 0;
+    transition: all 0.2s ease;
+}
+
+.navbar .site-name {
+    display: none;    
+}
+
+.search-box {
+  order: 1;
+  margin-right: 0;
+  margin-left: 1rem;
+
+  .suggestions {
+    left: auto !important;
+    right: 0 !important;
+  }
+}
+
+.links .search-box input {
+    border: 1px solid #2C2C2C;
+    box-shadow: 0px 4px 4px #DCC3D3;
+    background: #FFF url(/assets/search.svg) 0.6rem 0.5rem no-repeat
+}
+
+.outbound {
+    display: none !important;    
+}
+
+ul.suggestions {
+    margin-top: 45px;
+}
+
+main.home {
+    padding: 10rem 2rem 0;
+}
+
+h1#main-title {
+    padding: 4rem 0;
+    padding-bottom: 3.5rem;
+    font-size: 34.59px;
+    font-family: 'Inter Bold', -apple-system, sans-serif;
+}
+
+.theme-default-content:not(.custom) > h1 {
+    padding-top: 9.5rem;
+}
+
+.theme-default-content:not(.custom) > h2,
+.theme-default-content:not(.custom) > h3,
+.theme-default-content:not(.custom) > h4,
+.theme-default-content:not(.custom) > h5 {
+    padding-top: 9.5rem;
+}
+
+aside.sidebar {
+    top: 8rem;
+}
+
+p.description {
+    display: none;
+}
+
+.main-menu {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 8rem;
+}
+
+.social-menu {
+    display: flex;
+    margin-bottom: 8rem;
+}
+
+.etc-menu {
+    display: block;
+    margin-bottom: 8rem;
+}
+
+// .menu {
+//     display: grid;
+//     grid-auto-flow: column;
+//     grid-auto-columns: 2fr 2fr 2fr 2fr;
+
+//     width: 800px;
+//     margin: 0 auto;    
+// }
+
+// .menu > * { flex: 1; }
+
+.main-menu_container {
+    display: flex;
+    flex-direction: row;
+    flex: 1;
+    flex-basis: 50%;
+
+}
+
+.social-menu_container {
+    display: flex;
+    flex-direction: row;
+}
+
+@media screen and (max-width: 1179px) { 
+    .main-menu_container {
+        display: flex;
+        flex-direction: row;
+        flex: 1;
+        flex-basis: 100%;
+    }
+    .social-menu_container {
+        display: flex;
+        flex-direction: row;
+    }
+}
+
+.menu-items {
+    padding: .5em;
+    width: auto;
+}
+
+// Sidebar
+section .sidebar-heading {
+    color: $textColor;
+    font-size: 16px;
+}
+
+.sidebar-links a.sidebar-link {
+    color: $textColor;
+    font-size: 16px;
+}
+
+.sidebar-link .sidebar-sub-headers a.sidebar-link {
+    color: $thirdListTextColor;
+    font-size: 14px;
+}
+
+.sidebar-links li ul.sidebar-sub-headers .sidebar-sub-header a.sidebar-link {
+    // color: $thirdListTextColor;
+    font-size: 12px;
```

Comment:
**Some side notes:**
Do you want to keep the 4th list menu font size 14px or to 12px @joycehermosilla? 
Because we have 14px for 3rd list of menu fonts, we use 12px at the moment.

This is visual on it
<img width="255" alt="Screen Shot 2022-09-23 at 08 42 07" src="https://user-images.githubusercontent.com/20473526/191879590-0a6d23c7-dc98-4004-a56b-edf19c45ff2c.png">